### ISSUE TITLE
Add keyboard control to b-tables for better accessibility

### ISSUE THREAD

**Issue by tbl0605** [bootstrap-vue-next/bootstrap-vue-next#1995]
### Missing feature is from

bootstrap-vue

### Docs

https://bootstrap-vue.org/docs/reference/accessibility#accessibility

### Describe the missing feature

Hi,
first of all thank you for those great components!

### Describe the bug

After migrating our application from bootstrap-vue to bootstrap-vue-next, our users realized that they could no longer completely navigate the application using only the keyboard.
Our main issue is with the b-table component where it is not possible to select a row using tab, arrow and space keys.

### Reproduction

I provide 2 demos, where you can try to cycle through the components using the tab key (and using the space key inside the b-table to select a row):

- the boostrap-vue version where the keyboard accessibility works well: [demo 1](https://stackblitz.com/edit/vite-vue2-bootstrap-vue-totvxi)
- the bootstrap-vue-next version where the keyboard accessibility doesn't work on b-tables: [demo 2](https://stackblitz.com/edit/github-yk2gus-qufpri)

Btw, I didn't find a clear link to the bootstrap-vue documentation about b-table accessibility, keyboard accessibility is a more global feature across all bootstrap-vue components (see [bootstrap-vue accessibility](https://bootstrap-vue.org/docs/reference/accessibility#accessibility)). It would be nice if bootstrap-vue-next could do the same thing with all its components :)

### Validations

- [X] Have tested with the latest version. This is still alpha version and sometime things change rapidly.
- [x] Follow our [Code of Conduct](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided link is to the documentation of the missing feature.

**Comment by VividLemon** [comment]
Duplicate of https://github.com/bootstrap-vue-next/bootstrap-vue-next/issues/1259
